### PR TITLE
Notes reminder refactor

### PR DESCRIPTION
**Pull request by Saif-Shines**

### CODE REVIEW

File: Demo-Apps/notes_reminder/server/server.js [hunk 1]
```diff
@@ -1,59 +1,61 @@
...
           renderData(null);
         },
-        function(err) {
+        function operationErr(err) {
           renderData({
-            message: "Schedule creation failed"
+            message: 'Schedule creation failed',
           });
-        }
+        },
```
Comment (ravirajsubramanian) [inline Demo-Apps/notes_reminder/server/server.js:58]: Is this comma required at the end?

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (ravirajsubramanian) [inline .eslintrc:1]: Please remove this file and add it to the root of the repo if required.

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (ravirajsubramanian) [inline .prettierrc:1]: Please remove this file and add it to the root of the repo if required.

File: Demo-Apps/notes_reminder/server/server.js (hunk 1 above)
Comment (ravirajsubramanian) [reply Demo-Apps/notes_reminder/server/server.js]: Looks like it has been removed yet.

Comment (Saif-Shines) [reply Demo-Apps/notes_reminder/server/server.js]: Addressed it.

File: .eslintrc (hunk 2 above)
Comment (Saif-Shines) [reply .eslintrc]: This file is not intended to be part of this PR. Removed.

**Pull request closed without merging**